<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorFillingGameClass</class>
 <widget class="QMainWindow" name="ColorFillingGameClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1118</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorFillingGame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_show_img">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>800</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_resolution">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>820</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> aa </string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>218</width>
      <height>270</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_reset">
       <property name="text">
        <string>恢复</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_threshold">
         <property name="text">
          <string>二值化</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_threshold_value"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_adaptive_threshold">
       <property name="text">
        <string>局部阈值化</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_remove_small_centroids">
       <property name="text">
        <string>清除杂点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_colors" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1118</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ColorFillingGame.qrc"/>
 </resources>
 <connections/>
</ui>
